### ISSUE TITLE
Softmax dimension is wrong.

### ISSUE THREAD

**Issue by hwanheelee1993** [batra-mlp-lab/visdial-challenge-starter-pytorch#1]
I think there is critical mistake in decoder code.

Dimension for implementing Softmax is wrong. 

Since the score tensor's size is [batch_size x answer_options],  dimension should be changed 
(dim 0 -> dim 1)

**Comment by kdexd** [comment]
Thanks for pointing out the typo! Fixed in `07d8117`
